app/controllers/orders_controller.rb

```diff
@@ -19,7 +22,13 @@ def create
           })
       end
       session.delete(:cart)
-      redirect_to "/orders/#{order.id}"
+      if current_user
```

Comment:
Do we need to make sure that it is a user here? Maybe it would be better to do this at the beginning of the method